lib/tasks/create_application.rake

```diff
@@ -1,40 +1,30 @@
 # frozen_string_literal: true
 
-# NB: there is an issue in Doorkeeper which will cause an "uninitialized constant Doorkeeper::Application" error when
-# running this rake task in production mode. It relates to the lazy-loading of ActiveRecord and the work-around is to call
-# ActiveRecord::Base.logger first. See https://github.com/doorkeeper-gem/doorkeeper/issues/1319
-ActiveRecord::Base.logger
-
 namespace :auth do
-  NO_OWNER = 'No owner'
-
   desc 'Create an OAuth2 authorised client application'
   task create_client_application: :environment do
     require 'doorkeeper/orm/active_record/application'
-    require 'tty-prompt'
-
-    prompt = TTY::Prompt.new
 
-    name = prompt.ask("What's the application's name?")
+    puts "What's the application's name?"
+    name = STDIN.gets.chomp
 
     application = Doorkeeper::Application.new(name: name)
 
     suppliers = Supplier.all.each_with_object({}) { |supplier, accumulator|
-      accumulator[supplier.name] = supplier.id
-    }.merge(NO_OWNER => nil)
-    owner_id = prompt.select('Is this application owned by a supplier?', suppliers)
-    if owner_id != NO_OWNER
-      owner = Supplier.find(owner_id)
-      application.owner = owner
-    end
+      accumulator[supplier.name.downcase] = supplier.id
```

Comment:
Downcasing to make it easier to type in the CLI

Reply:
no need for downcase: you can do `supplier.key` instead, e.g. 
```
suppliers = Supplier.all.map{|s| [s.key,  s.id]}.to_h.merge('none' => nil)
```